main.go

```diff
@@ -112,17 +116,36 @@ func readStdin() string {
 	return b.String()
 }
 
+// defaultConfig returns the default config path
+func defaultConfig() string {
+	if c := os.Getenv("NAP_CONFIG"); c != "" {
+		return c
+	}
+	cfgPath, err := xdg.ConfigFile("nap/config.yaml")
+	if err != nil {
+		return "config.yaml"
+	}
+	return cfgPath
+}
+
 // readConfig returns a configuration read from the environment.
 func readConfig() Config {
-	config := Config{Home: defaultHome()}
-	if err := env.Parse(&config); err != nil {
-		return Config{
-			Home:            defaultHome(),
-			File:            defaultSnippetFileName,
```

Comment:
This is where I grabbed the default snippet filename from, but after review I think you're correct it should be `snippets.json`

Reply:
Yeah sorry about the confusion, I think that was incorrect originially but the `envDefault` was overriding it.